invenio_rdm_migrator/load/ids.py

```diff
@@ -32,10 +32,32 @@ def pid_pk():
 
 
 def generate_pk(data):
-    """Generate a primery key."""
+    """Generate a primary key."""
     return pid_pk()
 
 
+def generate_pk_for(model):
```

Comment:
all the `generate_...` functions were getting `data` in the signature due to the logic of the load class and where the PKs are generated (i.e. otherwise it might fail to create them) but I guess this was not the case since tests are passing.

Reply:
This function generates and returns a proper `generate_...` function; the inner returned function does follow the convention with the `data` parameter :)

Reply:
yes, it's all ok. I have the habit of adding comments along the review when I see something that I might fall again and doubt. So if you see "note to self" feel free to ignore, I can also stop adding them; thought it might be useful for other reviewers.

Reply:
hehe, no worries, I saw the original comment and started typing quickly :)